admin/src/components/drawer/SideBar.tsx

```diff
@@ -57,7 +57,7 @@ const SideBar = ({ sessionsList }) => {
 
   //State
   const [open, setOpen] = React.useState(false);
-  const test = useSelector((state: AuthState) => state.Auth.previousSigninToken);
+  const isImpersonate = localStorage.getItem("isImpersonate");
```

Comment:
```suggestion
  const isImpersonate = !!user.impersonateId;
```
se baser sur le `user`